components/composed/contribution/CollectionContributionList/CollectionContributionList.tsx

```diff
@@ -1,28 +1,32 @@
 import React from "react";
-import ModelListPage from "components/composed/model/ModelListPage";
-import { OperationType } from "relay-runtime";
-import { graphql, useFragment } from "react-relay";
-import {
+import { graphql } from "react-relay";
+import type { OperationType } from "relay-runtime";
+import type {
   CollectionContributionListFragment,
   CollectionContributionListFragment$key,
 } from "@/relay/CollectionContributionListFragment.graphql";
-import ModelColumns from "components/composed/model/ModelColumns";
-import { DataViewOptions } from "components/atomic/DataViewToggle";
 import type { ModelTableActionProps } from "react-table";
 import { CellProps } from "react-table";
-import { NamedLink } from "components/atomic";
 import { useRouter } from "next/router";
+import { useMaybeFragment } from "hooks";
 
-interface CollectionContributionListProps {
-  data: CollectionContributionListFragment$key;
-}
+import { NamedLink } from "components/atomic";
```

Comment:
Out of curiosity, do you think we should add a lint rule for import ordering? I get frustrated when they're out of order, or when components are above a react import... is that just me, or do other folks get annoyed by it too? 😅

Reply:
Yeah, I would love a rule, because I don't really know what order they're supposed to be in other than "react first"